src/dev/ci_setup/get_percy_env.js

```diff
@@ -25,5 +25,8 @@ const shortCommit = commit.slice(0, 8);
 
 const isPr = process.env.JOB_NAME.includes('elastic+kibana+pull-request');
 
+const { stdout: branch } = execa.sync('git', ['rev-parse', '--abbrev-ref', 'HEAD']);
+
 console.log(`export PERCY_PARALLEL_TOTAL=2;`);
 console.log(`export PERCY_PARALLEL_NONCE="${shortCommit}/${isPr ? 'PR' : pkg.branch}/${process.env.BUILD_ID}";`);
+console.log(`export PERCY_TARGET_BRANCH="${isPr ? process.env.ghprbTargetBranch : branch}";`);
```

Comment:
Where is `ghprbTargetBranch` coming from? Is there a reason we are using it over `PR_TARGET_BRANCH`?

Reply:
it's injected from the jenkins plugin, but +1 for PR_TARGET_BRANCH